autogen.sh

```diff
@@ -46,20 +46,6 @@ ProgHomeDir() {
     eval $2=$proghome
 }
 
-
-########################################################################
-echo
-echo "########################################################"
-echo "## Checking to make sure the directory structure is sane"
-echo "########################################################"
-echo
-
-if test ! -e "src/mpl/confdb/aclocal.m4" ; then
```

Comment:
I think this need to be changed to check for `hwloc` rather than being removed.

Reply:
I guess that should go in Hydra's `autogen.sh`. There is no `hwloc` in MPICH yet.

Reply:
Hmm, on second thought, it could probably go here.